LICENSE

```diff
@@ -323,42 +334,6 @@ library.  If this is what you want to do, use the GNU Library General Public
 License instead of this License.
 
 
-"CLASSPATH" EXCEPTION TO THE GPL
```

Comment:
External counsel advised this section can be removed. This section from Oracle is explaining which parts of their codebase is under GPL+CPE and how to identify such code ("but only where Oracle has expressly included in the particular source file's header the words "Oracle designates this particular file as subject to the "Classpath" exception as provided by Oracle in the LICENSE file that accompanied this code.")

Since all our patches in this repo are subject to GPL+CPE then we can simply remove it. If and when this is no longer the case and we do have patches of the other non-GPL+CPE components, then we will need to revert back and get clarification on needs to be added here.

Reply:
Strongly agree. This seems to be more relevant for the original code, not for the modification of it